src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -94,7 +150,8 @@ def is_runnable(self) -> bool:
             statements = parse_sql(self._original_code)
             return len(statements) > 0
         except SQLParseError:
-            return False
+            sqlglot_logger.warning(f"Failed to parse SQL using 'sqlglot': {self._original_code}")
```

Comment:
let's also log the sqlglot error, so that we can create issues over there.

Reply:
done